src/taipy/gui/utils/_evaluator.py

```diff
@@ -249,7 +266,7 @@ def re_evaluate_expr(self, gui: Gui, var_name: str) -> t.Set[str]:
             expr_decoded, _ = _variable_decode(expr)
             hash_expr = self.__expr_to_hash.get(expr, "UnknownExpr")
             if expr != var_name and not expr.startswith("_Taipy"):
-                expr_var_map = self.__expr_to_var_map.get(expr)  # ["x", "y"]
+                expr_var_map = self.__expr_to_var_map.get(expr, {})  # ["x", "y"]
                 if expr_var_map is None:
```

Comment:
How can this be None ?

Reply:
you are right, I will remove the empty dict assignment. I added that because mypy said expr_var_map has wrong type.